### PR TITLE
Filebeat: Add option to convert kafka module timezones to UTC

### LINKED ISSUE #7546: [beats] Kafka module - add convert_timezone for ES 6.1 and higher
Similar to https://github.com/elastic/beats/issues/3898

It is possible to [add timezone information](https://www.elastic.co/guide/en/beats/auditbeat/6.1/add-locale.html) already
However when using kafka module, the pipeline will not use this information unless pipeline is modified to add : 
```
"timezone": "{{ beat.timezone }}",
"ignore_failure": true
```

This enhancement request is to add feature similar to system module [var.convert_timezone](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-system.html#_literal_syslog_literal_fileset_settings)

### PR DESCRIPTION
**Pull request by jsoriano**
Fixes #7546

### CODE REVIEW

File: filebeat/module/kafka/log/manifest.yml [hunk 1]
```diff
@@ -9,6 +9,13 @@ var:
       - "{{.kafka_home}}/logs/server.log*"
       - "{{.kafka_home}}/logs/state-change.log*"
       - "{{.kafka_home}}/logs/kafka-*.log*"
+  - name: convert_timezone
+    default: false
+    # if ES < 6.1.0, this flag switches to false automatically when evaluating the
+    # pipeline
+    min_elasticsearch_version:
```
Comment (ruflin) [inline filebeat/module/kafka/log/manifest.yml]: I completely forgot we have this feature. Nice.

Comment (jsoriano) [reply filebeat/module/kafka/log/manifest.yml]: I copied this part from the system module :smile:

**Comment by jsoriano** [comment]
Rebased

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -216,6 +216,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Add Audit log fileset to the Elasticsearch module. {pull}7365[7365]
 - Add Slow log fileset to the Elasticsearch module. {pull}7473[7473]
 - Add deprecation fileset to the Elasticsearch module. {pull}7474[7474]
+- Add `config_timezone` option to Kafka module to convert dates to UTC. {issue}7546[7546] {pull}7578[7578]
```
Comment (ycombinator) [inline CHANGELOG.asciidoc]: @jsoriano I think this line should say `convert_timezone`, not `config_timezone`, correct?

Comment (jsoriano) [reply CHANGELOG.asciidoc]: Correct, I'll open a PR to fix this.

Comment (jsoriano) [reply CHANGELOG.asciidoc]: https://github.com/elastic/beats/pull/8001

**Pull request merged**